Jellyfin.Plugin.KodiSyncQueue/Jellyfin.Plugin.KodiSyncQueue.csproj

```diff
@@ -12,7 +12,8 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Jellyfin.Controller" Version="10.*" />
+    <PackageReference Include="Jellyfin.Data" Version="10.*-*" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.*-*" />
```

Comment:
Does changing this disallow normal package reference?

Reply:
No, from my quick testing, this matches both the `10.5.5` packages currently in the nuget repo, and the `10.6.0-datetime` format on the packages I built from master.

This pr can be used as a quick test for this;
https://github.com/jellyfin/jellyfin-plugin-isomounter/pull/3